### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by VaidehiSom**

### CODE REVIEW

File: scripts/mpc_test.py [hunk 1]
```diff
@@ -0,0 +1,1018 @@
...
+import time
+import subprocess
+import queue
+
+from setuptools import setup
+# from tkinter import _ImageSpec
+# from agents.navigation.controller import VehiclePIDController
+
+
+sys.path.append('../carla/dist/carla-0.9.9-py3.7-linux-x86_64.egg')
```
Comment (ayushgoel24) [inline scripts/mpc_test.py:13]: Will have to change path based on system

File: scripts/spawn_peds.py [hunk 2]
```diff
@@ -0,0 +1,549 @@
+# https://github.com/carla-simulator/carla/tree/master/PythonAPI/examples
+import glob
+import os
+import sys
+import time
+import subprocess
+
+sys.path.append('../carla/dist/carla-0.9.9-py3.7-linux-x86_64.egg')
```
Comment (ayushgoel24) [inline scripts/spawn_peds.py:8]: Path can be injected through properties file / YAML file

**Review by ayushgoel24 — APPROVED** [review]
LGTM

File: scripts/mpc_test.py (hunk 1 above)
Comment (VaidehiSom) [reply scripts/mpc_test.py:13]: Yup. Will do that

File: scripts/spawn_peds.py (hunk 2 above)
Comment (VaidehiSom) [reply scripts/spawn_peds.py:8]: Will update all this once we setup CARLA

Comment (ayushgoel24) [reply scripts/spawn_peds.py:8]: yup.. just mentioning this for future reference

**Pull request merged**